Firestore/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- [changed] `Transaction.getDocument()` now returns a non-nil
```

Comment:
Should we make this statement more strongly indicate that this is a breaking change and how to change to accommodate that?

For example, we could mention that for code that previously did `if (snap) ...` it should now do `if (snap.exists) ...`?